tests/integrationtests/tst_perfparser.cpp

```diff
@@ -262,6 +266,7 @@ private slots:
             QCOMPARE(parsingFinishedSpy.count(), 0);
             const auto message = qvariant_cast<QString>(parsingFailedSpy.takeFirst().at(0));
             QVERIFY(message.contains(errorMessagePart));
+            QVERIFY(message.contains(perfFile));
```

Comment:
dito, parent patch

Reply:
dito - not possible in previous patch as until with the "real patch" the filename was not part of the error message